pages/dashboard/views/index.jsx

```diff
@@ -1,21 +1,26 @@
 import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
+import Tasklist from '../../common/views/components/tasklist';
 import Link from '../../common/views/containers/link';
 import Snippet from '../../common/views/containers/snippet';
 
 const Index = ({
   profile: {
     firstName,
     establishments
-  }
+  },
+  tasks
 }) => (
   <Fragment>
     <header>
       <h2>&nbsp;</h2>
       <h1><Snippet name={firstName}>pages.dashboard.greeting</Snippet></h1>
     </header>
     <div className="govuk-grid-row">
-      <div className="govuk-grid-column-two-thirds">
+      <div className="govuk-grid-column-full">
+
+        <Tasklist tasks={tasks} />
```

Comment:
`tasks.length &&` <TaskList> ? to avoid rendering an empty task list if they don't have any?

Reply:
or perhaps we could handle it within the component - if there are no tasks then show a message

Reply:
It would also be worth using a connected component for TaskList rather than passing tasks, as the parent doesn't care about them and they have to be passed through...

```
// common/views/containers/task-list.js
import { connect } from 'react-redux';
import TaskList from '../components/task-list';

const mapStateToProps = ({ static: { tasks } }) => ({ tasks });

export default connect(mapStateToProps)(TaskList);
```

Reply:
this way `containers/task-list` is portable and can be used anywhere, as long as `tasks` are provided in locals

Reply:
The tasklist displays a message if there are no tasks, so that's already handled in the component.

The container makes sense, I'll stick it in.